src/no/runsafe/worldeditbridge/WorldEditCommands.java

```diff
@@ -65,8 +64,8 @@ public boolean regenerate(IPlayer runsafePlayer, ILocation pos1, ILocation pos2,
 			console.logException(e);
 			return false;
 		}
-		Mask mask = editSession.getMask();
-		editSession.setMask(null);
+		com.sk89q.worldedit.function.mask.Mask mask = editSession.getMask();
+		editSession.setMask((com.sk89q.worldedit.function.mask.Mask) null);

```

Comment:
Why is this being done? An import is being deleted and instead replaced with static inline package qualifiers. I can't see a purpose in this. Why not just keep the original import statement? Also casting NULL to a `Mask` seems weird. Review please.